apps/aeutils/priv/status

```diff
@@ -3,74 +3,55 @@
 
 -mode(compile).
 
--define(DEFAULT_FILEPATH, "aeternity_db.export").
-
 main(Args) ->
-    {RestArgs, TargetNode} = process_node_args(Args, [], undefined),
-    {Help, FilePath} = process_args(RestArgs),
-
-    maybe_show_help(Help),
-
-    connect_node(TargetNode),
-
-    do_main(TargetNode, FilePath).
-
-usage() ->
-    io:format("Usage: export_chain [help | FILE_PATH]~n", []).
-
-error_exit(Fmt, FmtArgs) ->
-    io:format(Fmt ++ "~n", FmtArgs),
-    halt(1).
-
-do_main(TargetNode, FilePath) ->
-    case filelib:is_regular(FilePath) of
-        true ->
-            error_exit("File ~s already exists", [FilePath]);
-        _ ->
-            mkdir_p(filename:dirname(FilePath)),
-            case rpc:call(TargetNode, aeu_export, to_disklog, [FilePath]) of
-                {ok, BlockCount} ->
-                    io:format("~i blocks exported to ~s~n", [BlockCount, FilePath]),
-                    halt(0);
-                {error, Reason} ->
-                    error_exit("Error while exporting to ~s: ~p", [FilePath, Reason])
-            end
+    {_RestArgs, TargetNode} = process_node_args(Args, [], undefined),
```

Comment:
This is a legacy script, so it doesn't use the `argparse` model and `aeu_ext_scripts` helpers like, e.g. the `db_rollback` script does. Granted, the benefits for this particular script would be limited, but it would look more consistent.